File: app/src/main/java/com/example/indroydlab/Data/ProductRepository.java
package com.example.indroydlab.Data;

import com.example.indroydlab.model.Product;
import java.util.ArrayList;
import java.util.List;

public class ProductRepository {
    public static List<Product> getProducts() {
        List<Product> products = new ArrayList<>();

        products.add(new Product(1, "Wireless Headphones", 2500, 1999.0, true, 18));
        products.add(new Product(2, "Bluetooth Speaker", 1500, null, false, 18));
        products.add(new Product(3, "Smart Watch", 3000, 2499.0, true, 18));
        products.add(new Product(4, "USB Cable", 400, null, false, 5));
        products.add(new Product(5, "Power Bank", 1200, null, false, 5));

        return products;
    }
}

File: app/src/main/java/com/example/indroydlab/data/ProductRepository.kt
package com.example.indroydlab.data

import com.example.indroydlab.model.ProductModel

object ProductRepository {

    fun getProducts(): List<ProductModel> {
        return listOf(
            ProductModel(1, " wireless Headphone", 2500.0, 1999.0, true, 18.0),
            ProductModel(2, "Bluetooth Speaker", 1500.0, null, false, 18.0),
            ProductModel(3, "Smart Watch", 3000.0, 2499.0, true, 18.0),
            ProductModel(4, "USB Cable", 400.0, null, false, 5.0),
            ProductModel(5, "Power Bank", 1200.0, null, false, 5.0),
            ProductModel(6, " wireless Headphone", 2500.0, 1999.0, true, 18.0)
        )
    }
}
